src/main/resources/com/linecorp/armeria/server/docs/assets/css/armeria.css

```diff
@@ -148,4 +157,15 @@ h2 {
 
 .debug-textarea {
   width: 100%;
+  max-width: 100%;
```

Comment:
This happens without max-width in Opera Beta which is basically same with Chrome:

![image](https://cloud.githubusercontent.com/assets/173918/11453047/61eeca28-963f-11e5-9d4a-2440310ed089.png)